### DISCUSSION TITLE
Select a portion of colorbar other than using mouse

### DISCUSSION THREAD

**Discussion by dev-ltk** [firemodels/fds#13099] [General]
Is there any ways other than using a mouse to select a portion of colorbar? For example, using a script or ini file or even introducing a new input field. It will help in automating the postprocessing process.

There was a similar discussion before, but yet not getting any update.
https://groups.google.com/g/fds-smv/c/a2gfY3ytPvk/m/Xxw2kesaAwAJ

**Comment by gforney** [comment]
with the existing smokeview you can edit the .ini file to set the colorbar selection width and location.  there are other parameters but edit i3 to set the color bar selection location and i4 to set the width
COLORBAR
 i1 i2 i3 i4

with the new windows smokeview posted at URL below you can set the color bar selection width (you could do this before) and location (new).  see attached dialog.

smokeview url: https://github.com/gforney/test_bundles/releases/tag/SMOKEVIEW_TEST

![colorbar_dialog](https://github.com/firemodels/fds/assets/12403014/e1ac23d4-b773-40b0-8584-5376f44df179)

**Comment by gforney** [comment]
also, if you select -1 for the selection index the select region will go away (this is the default)

this smokeview will show up in the nightly bundles, the next time we have a firebot pass

**Comment by dev-ltk** [comment]
The 'selection index' field in the new window does work.

However, when I load the the .ini file with i3 and i4 set, the selected region does not appear on the visualized model.

**Comment by gforney** [comment]
try smokeview posted here
https://github.com/gforney/test_bundles/releases/tag/SMOKEVIEW_TEST

**Comment by dev-ltk** [comment]
Tried the smokeview version above. It is still not working as intended. While I have saved a configuration file and checked that i3 of COLORBAR has been set, the selected portion does not appear once I reopen the smokeview and reload the configuration file.

**Comment by gforney** [comment]
its working for my case.  upload your input file (change .fds to .txt) and your .ini file and i'll take a look to see if there is something different that is causing a problem

**Comment by gforney** [comment]
I made some corrections which may help, try smokeview posted here (it's a different smokeview than posted first time at this link)
https://github.com/gforney/test_bundles/releases/tag/SMOKEVIEW_TEST

**Reply by dev-ltk** [reply]
> I made some corrections which may help, try smokeview posted here (it's a different smokeview than posted first time at this link) https://github.com/gforney/test_bundles/releases/tag/SMOKEVIEW_TEST

I found that I have to remove the old version of smokeview completely and clean install this build to make it work.

I can confirm this build can read the .ini file and load the prescribed highlighted range in the colorbar.
